app/services/auth.service.ts

```
@@ -53,8 +73,22 @@ export async function login(data: LoginData): Promise<AuthResponse> {
+            localStorage.setItem('accessToken', resData.session.access_token);
+        }
+        if (resData.session.refresh_token) {
+            localStorage.setItem('refreshToken', resData.session.refresh_token);
+        }
+
+        return resData;
+
     } catch (error: unknown) {
-        throw new Error('Login failed');
+        console.error(error);
+        throw new Error('Login failed' + error);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)